src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Initializers/WebJobsTelemetryInitializer.cs

```diff
@@ -130,5 +132,69 @@ private static string GetRoleInstanceName()
 
             return instanceName;
         }
+
+        /// <summary>
+        /// Changes properties of the RequestTelemetry to match what Functions expects.
+        /// </summary>
+        /// <param name="request">The RequestTelemetry to update.</param>
+        private void UpdateRequestProperties(RequestTelemetry request)
+        {
+            request.Context.GetInternalContext().SdkVersion = _sdkVersion;
+
+            // If the code hasn't been set, it's not an HttpRequest (could be auto-tracked SB, etc).
+            // So we'll initialize it to 0
+            if (string.IsNullOrEmpty(request.ResponseCode))
+            {
+                request.ResponseCode = "0";
+            }
+
+            // If the Url is not null, it's an actual HttpRequest, as opposed to a
+            // Service Bus or other function invocation that we're tracking as a Request
+            if (request.Url != null)
+            {
+                // We don't want to include the query string in URLs
+                request.Url = new Uri(request.Url.GetLeftPart(UriPartial.Path));
+
+                // App Insights sets this as 'VERB /url/path'. We want to extract the VERB. 
```

Comment:
Is this documented and guaranteed not to change? can we really rely on this being the same?

Reply:
Per-version, yes. I have tests that validate it, so if we move forward and it breaks, we'll know and adjust. @lmolkova -- is there any other way for me to see the method?